external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/CachedKafkaConsumer.scala

```diff
@@ -45,9 +46,6 @@ private[kafka010] case class CachedKafkaConsumer private(
 
   private var consumer = createConsumer
 
-  /** indicates whether this consumer is in use or not */
-  private var inuse = true
```

Comment:
Thanks, good point !, 
According to guava cache docs, When cache "approaches" the size limit - "The cache will try to evict entries that haven't been used recently or very often. "